pkg/controller/pki/event_recorder.go

```diff
@@ -37,3 +37,7 @@ func (r *loggingRecorder) WithComponentSuffix(componentNameSuffix string) events
 func (r *loggingRecorder) ComponentName() string {
 	return "not implemented"
 }
+
+func (r *loggingRecorder) Shutdown() {
+	//no implemented
```

Comment:
Nit: I think it should be "noT implemented"

Reply:
oops typo, fixed